canvas_modules/harness/src/client/App.js

```diff
@@ -1242,10 +1242,13 @@ class App extends React.Component {
 			if (currentEditorNodeId && canvasController.getNode(currentEditorNodeId, activePipelineId)) {
 				commonPropertiesRef.applyPropertiesEditing(false);
 			}
+			let initialEditorSize;
```

Comment:
Instead of always passing this initialEditorSize in can we add a dropdown in the properties configuration panel that allows us to test with and without setting this?  I would create it with unset, small, medium, large.  When unset we shouldn't pass anything into the propertyInfo object.